bundler/lib/bundler/cli/exec.rb

```diff
@@ -69,20 +70,14 @@ def process_title(file, args)
     end
 
     def ruby_shebang?(file)
```

Comment:
I checked code usage this function is only used in deciding `load` or `exec`. is it really needed?

I mean obviously it will cause error if it's not a ruby script.

Reply:
`bundle exec` should be able to run non ruby script, and for that we need to `exec` code path, right?